tests/builder/help.rs

```diff
@@ -606,6 +606,33 @@ SUBCOMMANDS:
     sub     short about sub
 ";
 
+static SETUP_HELP_MESSAGE: &str = "test 1.3
+Kevin K.
+tests stuff
+
+USAGE:
+    test
+
+OPTIONS:
+    -h, --help       Print help information
+    -V, --version    Print version information
+";
+
+static SETUP_LONG_HELP_MESSAGE: &str = "test 1.3
+Kevin K.
+tests stuff
+
+USAGE:
+    test
+
+OPTIONS:
+    -h, --help
+            Print help information
+
+    -V, --version
+            Print version information
+";
```

Comment:
Nit; could you move these into the test function bodies?

A lot of older tests do this and the separation between the expected output and the rest of the test function makes it harder to follow.  We've been writing new tests with the expected string in the body and updating the old tests piecemeal.